app/assets/stylesheets/partials/_admin.css.scss

```diff
@@ -18,4 +18,22 @@
 
 .admin-left-navi-link-group-last {
   margin-bottom: 50px; // To make space to the bottom of the page
-}
\ No newline at end of file
+}
+
+// Matching Selectize widget to our look
+.locale-selection__input-widget.selectize-control {
+  .selectize-input {
+    border: 2px solid $border;
+    @include border-radius(em(5));
+    font-size: 16px;
```

Comment:
We have two interesting functions to deal with sizes:
em: calculates em value for given pixels
lines: calculates line-heights in relation to our default (24px)
ks. type.scss